src/python/pants/option/global_options.py

```diff
@@ -289,6 +289,12 @@ def register_bootstrap_options(cls, register):
                   'start up all concurrent invocations (e.g. in other terminals) without pantsd. '
                   'Enabling this option requires parallel pants invocations to block on the first')
 
+    # Calling pants command (inner run) from other pants command is not the intended behaviour.
+    # Pants commands with this option set don't use pantsd.
```

Comment:
```suggestion
    # Currently, pants commands with this option set don't use pantsd, but this effect should not be relied upon.
```

We don't want people to start setting this flag in order to disable pantsd as a side effect; if we're going to write that it currently has this effect, we should make it clear that that may change in the future.